daffodil-runtime2/src/main/resources/examples/NestedUnion.c

```diff
@@ -361,7 +361,8 @@ data_parseSelf(data *instance, PState *pstate)
         bar_parseSelf(&instance->bar, pstate);
         break;
     default:
-        pstate->error_msg = "node's _choice field was not initialized during parsing";
+        pstate->error_msg =
+            "Parse error: no match between choice dispatch key and any branch key";
```

Comment:
I think this should just abort(msg)

Reply:
Ah. I think I'm incorrect here. This is parse, so we're determining if there is a choice branch key match here. So this is a regular Processing/Parse error.